src/NServiceBus.Gateway.AcceptanceTests/When_sending_a_message_via_the_gateway.cs

```diff
@@ -18,9 +18,8 @@ public async Task Should_process_message()
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<Headquarters>(b => b.When(bus =>
                 {
-#pragma warning disable DE0003 // API is deprecated
```

Comment:
FYI these warnings come from the Microsoft.DotNet.Analyzers.Compatibility package, which does recommend not using `HttpWebRequest` : https://github.com/dotnet/platform-compat/blob/master/docs/DE0003.md

However, it doesn't look like we've added that package to this repo yet, so we don't have this warning yet.

Reply:
should we switch to start using `HttpClient` in the gateway then? Sounds like something to consider for a maintenance release?

Reply:
Raise https://github.com/Particular/NServiceBus.Gateway/issues/105